node-zerox/src/index.ts

```diff
@@ -43,16 +44,35 @@ export const zerox = async ({
   // Download the PDF. Get file name.
   const localPath = await downloadFile({ filePath, tempDir: tempDirectory });
   if (!localPath) throw "Failed to save file to local drive";
+
+  const fileExtension = path.extname(localPath).toLowerCase();
+  let pdfPath: string;
+  const supportedExtensions = [".docx", ".pptx", ".ppt"];
```

Comment:
Libreoffice support all file type conversions, so I think we can remove the `supportedExtensions`. 

What do you think of the below?
```
if (!fileExtension) {
    throw new Error("File extension missing");
  }
  
  
if (fileExtension !== ".png") {
    if (fileExtension === ".pdf") {
      pdfPath = localPath;
    } else {
      pdfPath = await convertFileToPdf({
        localPath,
        tempDir: tempDirectory,
        extension: fileExtension,
      });
    } 
    
    ...
```

Reply:
looks good